### PR TITLE
readme: update shields

### PR DESCRIPTION
**Pull request by umarcor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@
 **toolchain you should start with the [f4pga-examples repository](https://github.com/chipsalliance/f4pga-examples).**
 
 <p align="center">
-  <a title="License Status" href="https://github.com/SymbiFlow/f4pga-arch-defs/blob/master/COPYING"><img alt="License Status" src="https://img.shields.io/github/license/SymbiFlow/f4pga-arch-defs?longCache=true&style=flat-square"></a><!--
+  <a title="License Status" href="https://github.com/SymbiFlow/f4pga-arch-defs/blob/master/COPYING"><img alt="License Status" src="https://img.shields.io/github/license/SymbiFlow/f4pga-arch-defs?longCache=true&style=flat-square&label=License"></a><!--
   -->
-  <a title="Documentation Status" href="https://f4pga.readthedocs.io/projects/arch-defs/"><img alt="'Test' workflow Status" src="https://img.shields.io/readthedocs/symbiflow-arch-defs/latest?longCache=true&style=flat-square&logo=ReadTheDocs&logoColor=fff"></a><!--
+  <a title="Documentation Status" href="https://f4pga.readthedocs.io/projects/arch-defs/"><img alt="Documentation Status" src="https://img.shields.io/readthedocs/symbiflow-arch-defs/latest?longCache=true&style=flat-square&logo=ReadTheDocs&logoColor=fff&label=RTD"></a><!--
```
Comment (mithro) [inline README.md:9]: Most people won't know what "RTD" stands for, can use "Docs" instead?

File: README.md [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
   -->
-  <a title="'Tests' workflow Status" href="https://github.com/SymbiFlow/f4pga-arch-defs/actions?query=workflow%3ATests"><img alt="'Tests' workflow Status" src="https://img.shields.io/github/workflow/status/SymbiFlow/f4pga-arch-defs/Tests/master?longCache=true&style=flat-square&label=Test&logo=github%20actions&logoColor=fff"></a><!--
+  <a title="'Automerge' workflow Status" href="https://github.com/SymbiFlow/f4pga-arch-defs/actions/workflows/Automerge.yml"><img alt="'Tests' workflow Status" src="https://img.shields.io/github/workflow/status/SymbiFlow/f4pga-arch-defs/Automerge/main?longCache=true&style=flat-square&label=Actions&logo=github%20actions&logoColor=fff"></a><!--
```
Comment (mithro) [inline README.md:11]: Most people won't know what "Actions" means, can we continue to use "Tests" or something like that?

File: README.md (hunk 1 above)
Comment (umarcor) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (umarcor) [reply README.md]: Done.

**Review by mithro — APPROVED** [review]
LGTM!

**Comment by mithro** [comment]
I'm assuming the failing CI is unrelated to this change?

**Comment by umarcor** [comment]
> I'm assuming the failing CI is unrelated to this change?

I thin so.

**Pull request merged**